style.css

```diff
@@ -307,14 +302,125 @@ section {
   border-radius: 0.5rem;
 }
 
-.about-myself--speciality .programming-language span {
+.about-myself--speciality--option span {
   font-weight: 500;
   font-size: 15px;
   line-height: 20px;
   color: #253858;
   margin-left: 1rem;
 }
 
+.contact-section {
+  background-color: #fff;
+}
+
+.contact {
+  background-color: #6070ff;
+  background-image: url("images/contact-form-shapes-mobile.png");
+  background-size: auto 780px;
+  background-position: right;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  padding: 6.43rem 1.5rem 0 1.5rem;
+  border-top-left-radius: 100px;
+}
+
+.contact h2 {
+  font-weight: 700;
+  font-size: 2.5em;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 0.75em;
+}
+
+.contact p {
+  font-weight: 400;
+  font-size: 1.25em;
+  line-height: 1.75em;
+  text-align: center;
+  color: #ebebff;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 3.62rem;
+}
+
+input,
+textarea {
+  background-color: #fff;
+  box-sizing: border-box;
+  padding: 0.93rem 2.3rem 0.93rem 1rem;
+  border-radius: 0.5rem;
+  border: 1px solid #cfd8dc;
+  margin-bottom: 1.124rem;
+}
+
+input {
+  height: 3rem;
+  outline: none;
+  font-weight: 400;
+  font-size: 1.06em;
+  line-height: 1.25em;
+  color: #172b4d;
+}
+
+textarea {
+  height: 10.6rem;
+  outline: none;
+  font-weight: 400;
+  font-size: 0.937em;
+  line-height: 1.5rem;
+  color: #b3bac5;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly I suggest you remove the color: #b3bac5; so that when you will write your text in your text area it will be more visible. When I try to input the text it takes the grey color which is not visible please fix that  :+1: 

![image](https://user-images.githubusercontent.com/95297251/213030182-69a2e937-c09d-4c8a-8549-53b2a7939471.png)

Reply:
I will correct that ,Thanks for the review.